README.md

```diff
@@ -94,7 +85,42 @@ running in one terminal and run these commands in another.
 
 A helper script is included to start up the external DB to run a manage.py command. Simply
 replace `./manage.py` with `./extmanage.sh` to run a command with the DB running, and automatically
-clean up afterwards.
+clean up afterwards. For example, to get started, you would do the following steps:
+
+    ./extmanage.sh migrate
+    ./extmanage.sh createsuperuser
+    ./extmanage.sh runserver
+
+### Local development without an external drive
+
+If you'd prefer not to use an external drive, you may be able to instead use small slices of the
+voter files. You will not be able to reproduce the ability of the app to fetch and process full
+files, but it may be enough for you to meaningfully work on the application. In order to do this,
+download this [zipfile of "sliced" voter
+files](https://drive.google.com/file/d/1mc6cSFV5eG533GqAjsJiCyiZdnJ8fzbB/view?usp=sharing).
+
+Unzip the zipfile and record the directory name:
+
+    unzip slices.zip
+    cd slices
```

Comment:
Is there a need to cd into the slices directory?

Reply:
oops, no :)